client/src/apiClient.ts

```diff
@@ -0,0 +1,13 @@
+import createClient from "openapi-fetch";
```

Comment:
tady to `openapi-fetch` mas uveden v server/package.json, ale ten soubor je v klientovi, takze by to melo byt v client/package.json a jeho dependencich